src/sparseml/transformers/utils/helpers.py

```diff
@@ -235,7 +235,8 @@ def resolve_sequence_length(config: AutoConfig) -> int:
 
 
 def resolve_recipe(
```

Comment:
Would you expect there to be any external usage of this function?

Reply:
I would be surprised. It is not user-facing. Matter of fact, the only usage of it in the `src` happens within the `SparseAutoModelForCausalLM`